### DISCUSSION TITLE
Release 7.3.8 being installed, not listed

### DISCUSSION THREAD

**Discussion by philmiller84** [PowerShell/PowerShell#20467] [Q&A]
Hi, I'm seeing an unlisted release being installed in my Windows Updates. Is this something to be concerned about? 

![unlisted_release](https://github.com/PowerShell/PowerShell/assets/7882837/224b5cb6-4d89-4678-9ff0-374520e7b731)

**Comment by dbramblett** [comment]
Ditto, already installed it and couldn't find any information about 7.3.8 when I went looking for version history to see what changed.

**Comment by philmiller84** [comment]
Seems the release notes are up as of 3 hours ago
